processor.go

```diff
@@ -523,10 +523,27 @@ func (g *Processor) newStorage(topic string, id int32, update UpdateCallback) (*
 		}, nil
 	}
 
-	st, err := g.opts.builders.storage(topic, id)
+	wg := new(sync.WaitGroup)
+	var err error
+	var st storage.Storage
+	ticker := time.NewTicker(5 * time.Minute)
+	defer ticker.Stop()
+	go func() {
+		wg.Add(1)
```

Comment:
this wg.Add(1) needs to be outside of the go-routine, otherwise you might be super super unlucky and the outer go-routine might come to the wg.Wait()-call before the go-routine has started and won't wait then.